tests/withdrawals_test.go

```diff
@@ -0,0 +1,57 @@
+package tests
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/stretchr/testify/require"
+)
+
+func TestWithdrawalFieldsInBlocks(t *testing.T) {
```

Comment:
If moving RLP encoding tests out of this function, this test does not need to declare nested cases.

Reply:
RLP encode/decode actually needs the block. in that test case we want to check that blocks coming from the network are healthy to be encoded/decoded. I will change the test case name to reflect that.